.github/workflows/wheels.yml

```diff
@@ -32,16 +40,42 @@ jobs:
         os:
           - ubuntu-latest
           - macos-latest
-          # - windows-latest
+          - windows-latest
 
     steps:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
 
+    - name: Cache Conan dependencies
+      id: cache-conan
+      uses: actions/cache@v2
+      if: matrix.os == 'windows-latest'
+      with:
+        path:
+          $USERPROFILE/.conan
+          conan-build
+        key: conan-${{ hashFiles('conanfile.txt') }}-v1.1
+
+    - name: Set up Windows Python ${{ matrix.python_version }}
+      uses: actions/setup-python@v2
+      if: matrix.os == 'windows-latest'
+      with:
+        python-version: ${{ matrix.python_version }}
+
+    - name: Install conan for Windows
+      if: matrix.os == 'windows-latest' && steps.cache-conan.outputs.cache-hit != 'true'
+      run: |
+        pip install pip --upgrade
+        pip install conan
+        conan profile new default --detect
+        conan profile update "settings.compiler=Visual Studio" default
+        conan profile update "settings.compiler.version=16" default
+        conan install --build=openssl --install-folder conan_build .
```

Comment:
github discarded it so again:

I decided to take this here instead of `CIBW_BUILD_BEFORE_ALL` or so, since we want control when we attempt to install the conan dependencies (only if there was a cache miss). I didn't see a really good way to pass that on to cibuildwheel so it wouldn't execute a conan install conditionally. maybe env vars or so, but anyways, I think it's still fairly transparent. not sure yet if there's some linking issues when this is actually installed via `pip` on a target machine. there's no `.dll`s in the wheels, but I guess since the build links statically, there's simply no dependencies a windows build needs. we'll see the latest once the issue with numpy is resolved.